setup.cfg

```diff
@@ -0,0 +1,118 @@
+# This file is used to configure your project.
+# Read more about the various options under:
+# http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
+
+[metadata]
+name = spin1_beam_model
+description = Radio-Interferometric Beam Models in Spin-Spherical Harmonic space
+author = Zachary Martinot
+author-email = zmarti@sas.upenn.edu
+license = mit
+long-description = file: README.rst
+long-description-content-type = text/x-rst; charset=UTF-8
+url = https://github.com/UPennEoR/spin1_beam_model
+project-urls =
+    Documentation = https://pyscaffold.org/
+# Change if running only on Windows, Mac or Linux (comma-separated)
+platforms = any
```

Comment:
I think the ssht dependency means that we're only compativle with Mac and Linux...

Reply:
Good catch. Fixed that.